### DISCUSSION TITLE
github copilot at enterprise level seat management

### DISCUSSION THREAD

**Discussion by adrian-puiu** [community/community#67070] [Enterprise]
### Select Topic Area

Question

### Body

Hello, 

Short question , how can we get the list of all users that are using copilot in our enterprise which have more then 300 organizations ?

**Comment by Akash1134** [comment]
Hi @adrian-puiu ,

Please review this [Copilot Doc](https://docs.github.com/en/enterprise-cloud@latest/copilot/managing-copilot-for-business/managing-access-for-copilot-for-business-in-your-organization#configuring-access-to-github-copilot-in-your-organization) and let us know if that helps.
